python/build-wheel-manylinux1.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+cat << EOF > "/usr/bin/nproc"
+#!/bin/bash
+echo 1
+EOF
+chmod +x /usr/bin/nproc
+
+mkdir .whl
+for PYTHON in cp27-cp27mu cp33-cp33m cp34-cp34m cp35-cp35m cp36-cp36m; do
+  rm -rf *
```

Comment:
It seemed that `git clean` wasn't sufficient so I used `rm *`but it looks like `-f -f` will do the job.